lib/checker/mochikit.coffee

```diff
@@ -90,7 +87,7 @@ mochikitRegexes = mochikitFunctions.map (fn) ->
 
 phpFileWithScriptTag = (name, lines) ->
   isPhpFile = /\.php$/.test(name)
-  isPhpFile && lines.filter((line) -> /<script|js =/.test(line)).length > 0
+  isPhpFile && lines.filter((line) -> /<script|js|<<<EOF =/.test(line)).length > 0
```

Comment:
This is the other (horrible) way JS is included in the PHP - see for e.g. line 81 of htdocs/artefact/file/profileicons.php